mmv1/third_party/terraform/resources/resource_storage_transfer_job.go

```diff
@@ -443,6 +443,11 @@ func awsS3DataSchema() *schema.Resource {
 				Type:        schema.TypeString,
 				Description: `S3 Bucket name.`,
 			},
+			"path": {
+				Required:    false,
```

Comment:
`Optional:    true,` is used normally.

Reply:
i modified https://github.com/GoogleCloudPlatform/magic-modules/pull/7932/commits/9fc1675cca1d502265d038847d9f6769ce21b80d, thanks